backend/django/core/management/commands/seed_SMART.py

```diff
@@ -100,15 +100,15 @@ def handle(self, *args, **options):
                 name='Root Only Project',
                 description="This is a project for only the root user. The root user is the creator.  This project's data file has labels.",
                 data_file='./core/data/test_files/test_some_labels.csv',
-                label_list=['FAVOR', 'AGAINST', 'NONE'],
+                label_list=['about cats', 'not about cats', 'unclear'],
                 perm_list=[],
                 classifier='logistic regression')
             multi_user_project = seed_project(
                 creator=root,
                 name='Three User Project',
                 description="This is a project that is coded by three different users.  No labels were in the data file to begin with.",
                 data_file='./core/data/test_files/test_no_labels.csv',
-                label_list=['Good', 'Bad'],
+                label_list=['about cats', 'not about cats', 'unclear'],
                 perm_list=[user1, test_user],
                 classifier='logistic regression')
             seed_project(
```

Comment:
This last project needs to have the new set of labels too

Reply:
done